aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -113,10 +113,11 @@ public void initialize(String name, Map<String, String> properties) {
     this.catalogProperties = ImmutableMap.copyOf(properties);
     AwsClientFactory awsClientFactory;
     FileIO catalogFileIO;
-    if (PropertyUtil.propertyAsBoolean(
-        properties,
-        AwsProperties.GLUE_LAKEFORMATION_ENABLED,
-        AwsProperties.GLUE_LAKEFORMATION_ENABLED_DEFAULT)) {
+    if (isTableLevelTagEnabled(properties)) {
```

Comment:
why is this needed?

Reply:
Shouldn't be needed, I missed that `catalogProperties` was initialised as `ImmutableMap.copyOf(properties);`